### PR TITLE
Update docs regarding exporter_ -> exported_ prefix.

### PR DESCRIPTION
**Pull request by juliusv**

### CODE REVIEW

File: content/docs/concepts/jobs_instances.md [hunk 1]
```diff
@@ -26,10 +26,10 @@ scraped time series which serve to identify the scraped target:
 * `instance`: The `<host>:<port>` part of the target's URL that was scraped.
 
 If either of these labels are already present in the scraped data, Prometheus
-does not replace their values. Instead, it adds new labels with an `exporter_`
-prefix prepended to the label name: `exporter_job` and `exporter_instance`. The
-same pattern holds true for any labels that have been manually configured for a
-target group. This enables intermediary exporters to proxy metrics.
+prepends an `exported_` prefix to the existing label names (`exported_job` and
+`exported_instance`) and then attaches its own labels. The same pattern holds
+true for any labels that have been manually configured for a target group. This
```
Comment (brian-brazil) [inline content/docs/concepts/jobs_instances.md]: for any labels for a target

Labels coming from SD aren't necessarily manually configured.

**Comment by fabxc** [comment]
:+1:

File: content/docs/concepts/jobs_instances.md (hunk 1 above)
Comment (juliusv) [inline content/docs/concepts/jobs_instances.md]: Thanks, updated! Does that sound good?

File: content/docs/concepts/jobs_instances.md [hunk 2]
```diff
@@ -26,10 +26,10 @@ scraped time series which serve to identify the scraped target:
 * `instance`: The `<host>:<port>` part of the target's URL that was scraped.
 
 If either of these labels are already present in the scraped data, Prometheus
-does not replace their values. Instead, it adds new labels with an `exporter_`
-prefix prepended to the label name: `exporter_job` and `exporter_instance`. The
-same pattern holds true for any labels that have been manually configured for a
-target group. This enables intermediary exporters to proxy metrics.
+prepends an `exported_` prefix to the existing label names (`exported_job` and
+`exported_instance`) and then attaches its own labels. The same pattern holds
+true for any labels configured for a target. This enables intermediary
```
Comment (brian-brazil) [inline content/docs/concepts/jobs_instances.md]: The last sentence doesn't apply with the `honor_labels` change, I'd say remove it or mention `honor_labels` here.

Comment (msiebuhr) [inline content/docs/concepts/jobs_instances.md]: There's a way to switch it off? Should definitely be mentioned, as that would actually make `jobs` and `instance` labels useful wrt. proxies, IMHO.

**Comment by juliusv** [comment]
@brian-brazil @msiebuhr Check out https://github.com/prometheus/docs/pull/150, which aims to address these comments.

**Comment by juliusv** [comment]
Obsoleted by #150

**Pull request closed without merging**